packages/grafana-ui/src/types/datasource.ts

```diff
@@ -295,7 +313,7 @@ export interface ExploreStartPageProps {
  */
 export type LegacyResponseData = TimeSeries | TableData | any;
 
-export type DataQueryResponseData = DataFrameDTO | LegacyResponseData;
+export type DataQueryResponseData = DataFrame | DataFrameDTO | LegacyResponseData;
```

Comment:
unrelated... but I added it while I'm looking

Reply:
Shouldn't it be DataFrameHelper as well?

Reply:
no, `DataFrameHelper` implements `DataFrame` -- it is just a helper ;)